Documentation/deploy-workers.md

```diff
@@ -45,11 +44,11 @@ After=flanneld.service
 
 #### Create the kubelet Unit
 
-Create `/etc/systemd/system/kubelet.service` and replace: 
+Create `/etc/systemd/system/kubelet.service` and replace the following variables: 
 
-* Replace `${MASTER_IP}`
-* Replace `${ADVERTISE_IP}` with this nodes publicly routable IP.
-* Replace `${DNS_SERVICE_IP}`
+* `${MASTER_IP}`
+* `${ADVERTISE_IP}`, with this node's publicly routable IP.
+* `${DNS_SERVICE_IP}`
```

Comment:
That's actually not quite what I was referring to - rather that the intro to the section said "replace the following" then each item also said "replace x". Anyway I tweaked it to preserve your checklist :-)